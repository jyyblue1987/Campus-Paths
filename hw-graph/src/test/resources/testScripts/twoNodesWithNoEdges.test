# To test if we can have a graph with 2 nodes and no edges

# Create a graph
CreateGraph A

# Add two nodes
AddNode A n1
AddNode A n2

# Add 0 edges

# Print out the nodes in the graph along with their outgoing edges (if any)
ListNodes A
ListChildren A n1
ListChildren A n2