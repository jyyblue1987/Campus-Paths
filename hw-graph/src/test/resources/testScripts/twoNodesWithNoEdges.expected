# To test if we can have a graph with 2 nodes and no edges

# Create a graph
created graph A

# Add two nodes
added node n1 to A
added node n2 to A

# Add 0 edges

# Print out the nodes in the graph along with their outgoing edges (if any)
A contains: n1 n2
the children of n1 in A are:
the children of n2 in A are: